### PR TITLE
Automated recent items card group

### PR DESCRIPTION
**Pull request by rwd**
* Refactors cachers for DRY code
* Adds cacher for four recent items, one from each of the four most recently updated datasets containing content tier 4 items
* Cacher should be run periodically with `npm run cache items:recent set`
* Adds a server middleware to retrieve this cache
* Adds support for automatedCardGroup sections on browse pages, rendering RecentItems component for this section
* Included work-in-progress code for automated card group for featured topics, removed in commit 0e83854e to facilitate review, to be restored and completed in a follow-up branch

### CODE REVIEW

File: src/cachers/index.js [hunk 1]
```diff
@@ -2,18 +2,55 @@ if (process.env.NODE_ENV !== 'production') {
...
-const commands = [
-  'entities:organisations'
+const defu = require('defu');
+const utils = require('./utils');
+const nuxtConfig = require('../../nuxt.config');
+const runtimeConfig = defu(nuxtConfig.privateRuntimeConfig, nuxtConfig.publicRuntimeConfig);
+
+const cachers = [
+  'entities:organisations',
+  'entities:topics',
```
Comment (lbiedinger) [inline src/cachers/index.js:12]: this breaks the file on the current verion of the branch.
```
Error: Cannot find module './entities/topics'
```
Just commenting the line makes it work.

Comment (rwd) [reply src/cachers/index.js]: Indeed. It's been removed.

[the remark this answers is not in the document]
Comment (rwd) [reply src/cachers/index.js]: Yes, sorry, it's been removed.

File: src/components/browse/BrowsePage.vue [hunk 2]
```diff
@@ -54,6 +54,14 @@
...
         type: Object,
         default: null
+      },
+      recentItems: {
+        type: Boolean,
+        default: false
+      },
+      highlightedEntities: {
+        type: Boolean,
+        default: false
```
[the remark this answers is not in the document]
Comment (rwd) [reply src/components/browse/BrowsePage.vue]: Indeed not. Removed.

**Review by mirjamverloop — APPROVED** [review]

**Review by lbiedinger — CHANGES_REQUESTED** [review]

**Review by lbiedinger — APPROVED** [review]

**Pull request merged**